lib/serializer.js

```diff
@@ -38,3 +38,15 @@ Serializer.prototype.stringToByteArray = function (msg/* : string */) {
 Serializer.prototype.byteArrayToString = function (bytes/* : Uint8Array */) {
   return this.api.GenericMessage.decode(bytes).message
 }
+
+module.exports.serializer = null
+
+module.exports.initSerializer = function (apiFile/* : string */) {
```

Comment:
Updated this to return a Promise

maybe init() above should just do this?

i made initSerializer() so a whole app might share a single serializer instance, since it's kind of #effort to reinit the serializer each time we need it.